bridge/svix-bridge/src/webhook_receiver/mod.rs

```diff
@@ -37,8 +37,28 @@ fn router() -> Router<InternalState> {
             "/webhook/:integration_id/",
             post(route).put(route).get(route).patch(route),
         )
+        .route("/health", get(health_handler))
 }
+static START_TIME: once_cell::sync::Lazy<std::time::Instant> =
+    once_cell::sync::Lazy::new(|| std::time::Instant::now());
```

Comment:
To make clippy happy:

```suggestion
static START_TIME: once_cell::sync::Lazy<std::time::Instant> =
    once_cell::sync::Lazy::new(std::time::Instant::now);
```